playbooks/09-compliance.yml

```diff
@@ -0,0 +1,22 @@
+- name: Run compliance scan for controllers
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Run compliance scan for controllers
+      ansible.builtin.import_role:
+        name: compliance
+      vars:
+        cifmw_compliance_podman_username: "{{ cifmw_registry_token.credentials.username }}"
+        cifmw_compliance_podman_password: "{{ cifmw_registry_token.credentials.password }}"
+      when: cifmw_run_controller_compliance_scans | default('false') | bool
+
+- name: Run compliance scan for computes
+  hosts: "{{ groups['computes'] | default ([]) }}"
+  gather_facts: false
```

Comment:
I'd toggle this to `true` to ensure we really have the needed facts. Or call the `ansible.builtin.setup` with some `gather_subset: '!all,min'` or [something like that](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/setup_module.html#parameter-gather_subset).
For now, we do have facts because the framework still connects to the computes, but I wouldn't rely on that, really (we want to stop doing anything on the compute).

Reply:
will do